src/components/WorkspaceList/WorkspaceList.test.tsx

```diff
@@ -98,4 +102,44 @@ describe('Workspace list', () => {
       expect(screen.getByText('Delete workspace')).toBeTruthy();
     });
   });
+
+  test('list is accessible', async () => {
+    useK8sWatchResourceMock.mockReturnValue([[...workspacesMockData], true, null]);
+
+    const { container } = render(
+      <BrowserRouter>
+        <WorkspaceList />
+      </BrowserRouter>,
+    );
+    expect(await screen.findByText('demo-ws1')).toBeInTheDocument();
+
+    // NOTE this test will fail until https://github.com/openshift/dynamic-plugin-sdk/pull/175
+    // is merged and published
+    const results = await axe(container);
+    expect(results).toHaveNoViolations();
+  });
+
+  test('clicking on workspace name goes to details route', async () => {
+    useK8sWatchResourceMock.mockReturnValue([[...workspacesMockData], true, null]);
+
+    const history = createMemoryHistory();
+    history.push = jest.fn();
```

Comment:
Can we switch this to use `navigate` mock instead so we mirror what we have in the component itself.

Reply:
I'm not sure why, but changing this to just mock useNavigate isn't working.  Maybe because useNavigate is inside another component ???